### ISSUE TITLE
[Bug]: Compiler error when using Compose Auth Google or Apple composables in Kotlin/JS

### ISSUE THREAD

**Issue by jan-tennert** [supabase-community/supabase-kt#330]
### General Info

- [X] I checked for similar bug report
- [X] I am using the latest version
- [X] I checked the [troubleshooting](https://github.com/supabase-community/supabase-kt/wiki/Troubleshooting) page for similar problems
- [X] I enabled [logging](https://github.com/supabase-community/supabase-kt/wiki/Logging) and checked the logs

### Version(s)

1.3.X-1.4.X

### Target(s)

JS

### What happened? (include your code)

The Kotlin compiler will fail with an error when trying to use Compose Auth' composable functions within a composable. The problem is that both `rememberLoginWithGoogle` and `rememberLoginWithApple` are extension composable functions which the Kotlin/JS compiler doesn't like for some reason. A possible workaround is to wrap this function yourself:

```kotlin
//common
expect fun rememberGoogleLogin(composeAuth: ComposeAuth): NativeSignInState

//android
actual fun rememberGoogleLogin(composeAuth: ComposeAuth): NativeSignInState = composeAuth.rememberLoginWithGoogle()

//js
actual fun rememberGoogleLogin(composeAuth: ComposeAuth): NativeSignInState = defaultLoginBehavior { //this would be called by `rememberLoginWithGoogle`
    composeAuth.supabaseClient.gotrue.loginWith(Google)
}
```

### Steps To Reproduce (optional)

1. Add the Compose Auth to a Kotlin/JS project
2. Use either `composeAuth.rememberLoginWithGoogle` or `composeAuth.rememberLoginWithApple` within a composable
3. The compilation will not succeed

### Relevant log output (optional)

```shell
kotlin.NotImplementedError: Generation of stubs for class org.jetbrains.kotlin.ir.symbols.impl.IrValueParameterSymbolImpl is not supported yet
	at org.jetbrains.kotlin.backend.common.linkage.partial.MissingDeclarationStubGenerator.getDeclaration(MissingDeclarationStubGenerator.kt:63)
	at org.jetbrains.kotlin.backend.common.linkage.partial.PartiallyLinkedIrTreePatcher$ExpressionTransformer.checkReferencedDeclaration(PartiallyLinkedIrTreePatcher.kt:582)
	at org.jetbrains.kotlin.backend.common.linkage.partial.PartiallyLinkedIrTreePatcher$ExpressionTransformer.checkReferencedDeclaration$default(PartiallyLinkedIrTreePatcher.kt:574)
	at org.jetbrains.kotlin.backend.common.linkage.partial.PartiallyLinkedIrTreePatcher$ExpressionTransformer.visitDeclarationReference(PartiallyLinkedIrTreePatcher.kt:488)
	at org.jetbrains.kotlin.ir.visitors.IrElementTransformerVoid.visitValueAccess(IrElementTransformerVoid.kt:190)
	at org.jetbrains.kotlin.ir.visitors.IrElementTransformerVoid.visitGetValue(IrElementTransformerVoid.kt:193)
	at org.jetbrains.kotlin.ir.visitors.IrElementTransformerVoid.visitGetValue(IrElementTransformerVoid.kt:194)
	at org.jetbrains.kotlin.ir.visitors.IrElementTransformerVoid.visitGetValue(IrElementTransformerVoid.kt:24)
	at org.jetbrains.kotlin.ir.expressions.IrGetValue.accept(IrGetValue.kt:20)
	at org.jetbrains.kotlin.ir.expressions.IrExpression.transform(IrExpression.kt:31)
	at org.jetbrains.kotlin.ir.expressions.IrMemberAccessExpression.transformChildren(IrMemberAccessExpression.kt:49)
	at org.jetbrains.kotlin.ir.visitors.IrElementTransformerVoidKt.transformChildrenVoid(IrElementTransformerVoid.kt:346)
	at org.jetbrains.kotlin.backend.common.linkage.partial.PartiallyLinkedIrTreePatcher$ExpressionTransformer.visitCall(PartiallyLinkedIrTreePatcher.kt:1322)
```

**Comment by shubhamsinghshubham777** [comment]
@jan-tennert Can you check if this helps? https://github.com/JetBrains/compose-multiplatform/issues/4003#issuecomment-1846937685

**Comment by jan-tennert** [comment]
> @jan-tennert Can you check if these help?
> 
>     1. [kotlin.NotImplementedError: Generation of stubs for class org.jetbrains.kotlin.ir.symbols.impl.IrValueParameterSymbolImpl is not supported yet JetBrains/compose-multiplatform#4003 (comment)](https://github.com/JetBrains/compose-multiplatform/issues/4003#issuecomment-1846937685)
> 
>     2. https://youtrack.jetbrains.com/issue/KT-64257/K2-QG-kotlin.NotImplementedError-Generation-of-stubs-for-class

Yes, thanks! Seems like this issue has been fixed in Compose Multiplatform `1.6.0`.
